src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -276,19 +276,25 @@
     public ResponseEntity updateUserProfile(@Valid @RequestBody UpdateUserProfileData updateUserProfileData,
                                                                  @PathVariable String userId,
```

Comment:
Put @Pattern on each member of UpdateUserProfileData directly so that we need to call this method.

Reply:
This has already been implemented in the first commit: https://github.com/hmcts/rd-user-profile-api/pull/255/commits/12e2adfc2d6797551267395a1037442c6af6589d

I have removed the validator in the following commit: